File: proj1/fibo.c
#include <pthread.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#define MAX_FIB 3

int *fib_seq;
fib_seq = (int[2]){0, 1};

int main(void) {
	char buff[MAX_FIB];
	printf("How many fibonnaci numbers would you like? ");
	fflush(stdout);
	fgets(buff, sizeof(buff), stdin);
	buff[strcspn(buff, "\n")] = 0;
	int f = atoi(buff);
	printf("You entered: %d\n", f);
}

File: proj1/fibo.java
import java.util.Scanner;

class FibonacciGen {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.println("How many fibonacci numbers would you like?");
		String input = scanner.nextLine();
		int value = Integer.parseInt(input.trim());
	}
}